docs/weblog/README.md

```diff
@@ -41,6 +41,30 @@ The following text may be written to the body of the response:
 Hello headers!\n
 ```
 
+### GET /html
+
+This endpoint returns an HTML page instead of a JSON response. It is used to test features that require HTML content, such as RUM (Real User Monitoring) auto-injection.
+
+**Important:** This is a server-rendered HTML endpoint, not a REST/JSON endpoint. The tracer may inject additional scripts (like the RUM SDK) into the HTML response when RUM injection is enabled.
+
+The response content type must be `text/html`.
+
+A minimal HTML response structure:
```

Comment:
To ensure shared testing here, the endpoint **must** return the same content for all weblogs : 


```suggestion
The HTML content **must** be :
```